src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -16,7 +16,7 @@
   // public String cookieValue;
   public Cookie currCookie;
   private final String COOKIENAME = "sessionUserID";
-  private final String NO_LOGGED_IN_USER = "-1"; // -1 = logged out
+  public static final String NO_LOGGED_IN_USER = "-1"; // -1 = logged out
```

Comment:
nit:  I would prefer storing this as a long here, and do the string->long casting only in this class.  This way customers of this helper class wouldn't have to worry about casting.

A lot of the spots here that initializes with NO_LOGGED_IN_USER also probably doesn't need to start off with -1 (for example setLoggedInCookie can just set the userID off the bat), so that'll reduce the casts needed as well